cucumber/authenticators_jwt/features/authn_jwt_validate_restrictions.feature

```diff
@@ -0,0 +1,245 @@
+Feature: JWT Authenticator - Validate restrictions
+  Background:
+    Given I initialize JWKS endpoint with file "myJWKs.json"
+    And I load a policy:
+    """
+    - !policy
+      id: conjur/authn-jwt/raw
+      body:
+      - !webservice
+        annotations:
+          description: Authentication service for JWT tokens, based on raw JWKs.
+
+      - !variable
+        id: jwks-uri
+
+      - !variable
+        id: token-app-property
+
+      - !group users
+
+      - !permit
+        role: !group users
+        privilege: [ read, authenticate ]
+        resource: !webservice
+    """
+    And I am the super-user
+    And I successfully set authn-jwt jwks-uri variable with value of "myJWKs.json" endpoint
+
+  Scenario: General annotation invalid service specific annotation, 200 OK
+    Given I have a "variable" resource called "test-variable"
+    And I extend the policy with:
+    """
+    - !host
+      id: myapp
+      annotations:
+        authn-jwt/project-id: myproject
+        authn-jwt/aud: myaud
+        authn-jwt/raw/project-id: myproject
+        authn-jwt/raw/aud: myaud
```

Comment:
i suggest use different names for invalid service. **why?**
It will be more clear, if we will fail it will be better to understand on what.

Reply:
Chagned service name to invalid-service